x/msg_authorization/internal/keeper/keeper.go

```diff
@@ -0,0 +1,130 @@
+package keeper
```

Comment:
I believe having an `internal` folder was the old way of writing modules. Let's stick to having `keeper` at the root folder of the module